staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -204,12 +204,16 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"DEPRECATED: the namespace from which the Kubernetes master services should be injected into pods.")
 
 	fs.IntVar(&s.MaxRequestsInFlight, "max-requests-inflight", s.MaxRequestsInFlight, ""+
-		"The maximum number of non-mutating requests in flight at a given time. When the server exceeds this, "+
-		"it rejects requests. Zero for no limit.")
+		"This and --max-mutating-requests-inflight are summed to determine the server's total concurrency limit "+
+		"(which must be positive) if --enable-priority-and-fairness is enabled. "+
+		"Otherwise if this value is positive then: this flag gives the maximum number of non-mutating requests in flight "+
```

Comment:
For the second half, I suggest: "Otherwise, this flag gives the maximum number of non-mutating requests in flight, or a zero value disables the limit completely."

Reply:
(and I'd make the analogous suggestion for below, of course)

Reply:
done